### PR TITLE
Feature/cypress about page dw

### PR DESCRIPTION
**Pull request by devonwigle**
Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:

- Add test ids to AboutPage, MemberCard, ProfilePage, and TeamMembers components
- Wrote tests for about page and profile page user flows

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test:

- npm start
- new terminal window, npm run cypress
- observe tests running in about_page_spec.js

Additional comments:

### CODE REVIEW

**Comment by janitastic** [comment]
I cleaned up test ideas all over these files. We can simply put a test id on a top "section", and then use the `.children()` cypress attribute to call on other elements within a section/testid.

I also tested the click-ability of everything on the page. Did some fancy things with the social icons you all should check out if you're interested.

**Review by lschnell8 — APPROVED** [review]
Thanks for making those updates Jani! This looks much better.

File: cypress/integration/about_page_spec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  it('Should display a Meet the Team container with a title and instructions', () => {
+    cy.get('[data-testid=team-container]').should('be.visible').children()
+      .get('h3:nth-child(1)').contains('Meet the Team')
+      .get('[data-testid="team-container"] > p').should('have.length', 1)
+  })
+
+  it('Should display a grid of team member photos, names, pronouns, and social icons', () => {
+    cy.get('[data-testid=team-grid]').should('be.visible').children()
+      .get('[data-testid=member-card]').children()
```
Comment (cagallo) [inline cypress/integration/about_page_spec.js:41]: Children was a great way to go with this to DRY it up!

**Review by cagallo — APPROVED** [review]
Great job y'all, looks like a solid test

**Pull request merged**